jupyterlab_gitsync/deploy.sh

```diff
@@ -0,0 +1,13 @@
+INSTANCE_NAME=${1?}
```

Comment:
Can this be replaced with using `../scripts/deploy.sh` instead?

Reply:
I'm not entirely sure since this file was generated by the cookiecutter template that I used. I'm trying to keep as much of the template the same since the other extensions are also using this template.

Reply:
Yes you can drop this file as well. We've made some changes to the structure of the base repo since the cookiecutter was last updated. We can polish off some of those rough edges after this initial commit is in place.

Reply:
These files are removed in a future PR.